addon/services/feature-control-storage.js

```diff
@@ -0,0 +1,6 @@
+import Service from '@ember/service'
```

Comment:
Am I right saying that you created this Service, with only 1 property, just to make it easier to mock it during tests?

Reply:
Nope, the purpose of this property is to be read from `instance-initializers/load-feature-controls.js` & keeping consistency between read & write on local-storage....

In other words, I didn't find the way to properly take into account the `ember-local-storage` settings in `instance-initializers` when reading from local-storage without using this service...